packages/pg/lib/crypto/cert-signatures.js

```diff
@@ -18,11 +18,11 @@ function readASN1Length(data, index) {
 }
 
 function readASN1OID(data, index) {
-  if (data[index++] !== 0x6) x509Error('non-OID data', data) // 6 = OID
+  if (data[index++] !== 0x6) throw x509Error('non-OID data', data) // 6 = OID
 
   const { length: OIDLength, index: indexAfterOIDLength } = readASN1Length(data, index)
   index = indexAfterOIDLength
-  lastIndex = index + OIDLength
+  let lastIndex = index + OIDLength
```

Comment:
Yikes! Good catch.

Reply:
Could implications of this be interesting?  `lastIndex` might be scoped at the file/module level currently, but maybe safe in practice?

Reply:
I haven’t run it to check, but I assume it’s just leaking to the global object because the module isn’t in strict mode.